sherpa/cpp_api/brpc/server.cc

```diff
@@ -0,0 +1,378 @@
+/**
+ * Copyright      2022  (authors: Pingfeng Luo)
+ *
+ * See LICENSE for clarification regarding multiple authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <chrono>  //NOLINT
+#include <condition_variable>  //NOLINT
+#include <mutex>  //NOLINT
+#include <random>
+#include <shared_mutex>
+#include <thread>  //NOLINT
+#include <vector>
+
+#include "asr.pb.h" //NOLINT
+#include "sherpa/csrc/log.h"
+#include "sherpa/csrc/online_asr.h"
+#include "sherpa/csrc/parse_options.h"
+
+#include "brpc/server.h"
+#include "brpc/stream.h"
+#include "butil/base64.h"
+#include "butil/logging.h"
+
+using namespace std::chrono_literals;  //NOLINT
+
+const int32_t MAX_ASR_RESULT = 512;
```

Comment:
To follow the google code style, we usually use
`kMaxAsrResult` for a const variable.